### PR TITLE
Added Cleanup

### PR DESCRIPTION
**Pull request by ResurgenceDark**
- Error Fixes
- Cleanups

### CODE REVIEW

File: sku.0/sys.server/compiled/game/script/library/jedi.java [hunk 1]
```diff
@@ -2214,55 +2214,14 @@ public static void initializeCrystal(obj_id crystal, int level) throws Interrupt
     public static boolean isForceSensitive(obj_id player) throws InterruptedException
     {
         String classTemplate = getSkillTemplate(player);
-        if (classTemplate != null && classTemplate.startsWith("force"))
-        {
-            return true;
-        }
-        return false;
+        return (classTemplate != null && classTemplate.startsWith("force")) ? true : false;
```
Comment (Cekis) [inline sku.0/sys.server/compiled/game/script/library/jedi.java]: FYI, redundant...  you don't need logic on this at all.  Just return the stuff in the parentheses.

File: sku.0/sys.server/compiled/game/script/library/jedi.java [hunk 2]
```diff
@@ -2214,55 +2214,14 @@ public static void initializeCrystal(obj_id crystal, int level) throws Interrupt
...
     }
     public static boolean isForceSensitiveLevelRequired(obj_id player, int requiredLevel) throws InterruptedException
     {
-        int playerLevel = getLevel(player);
-        if (isForceSensitive(player) && playerLevel >= requiredLevel)
-        {
-            return true;
-        }
-        return false;
+        return (isForceSensitive(player) && getLevel(player) >= requiredLevel) ? true : false;
```
Comment (Cekis) [inline sku.0/sys.server/compiled/game/script/library/jedi.java]: Same as above.

File: sku.0/sys.server/compiled/game/script/library/jedi.java [hunk 3]
```diff
@@ -2214,55 +2214,14 @@ public static void initializeCrystal(obj_id crystal, int level) throws Interrupt
...
-        if (utils.playerHasStaticItemInAppearanceInventory(player, JEDI_CLOAK_LIGHT_HOOD_DOWN))
-        {
-            return true;
-        }
-        if (utils.playerHasStaticItemInAppearanceInventory(player, JEDI_CLOAK_DARK_HOOD_DOWN))
-        {
-            return true;
-        }
-        return false;
+        return (utils.playerHasStaticItemInBankOrInventory(player, JEDI_CLOAK_LIGHT_HOOD_UP) || utils.playerHasStaticItemInBankOrInventory(player, JEDI_CLOAK_DARK_HOOD_UP) || utils.playerHasStaticItemInBankOrInventory(player, JEDI_CLOAK_LIGHT_HOOD_DOWN) || utils.playerHasStaticItemInBankOrInventory(player, JEDI_CLOAK_DARK_HOOD_DOWN) || utils.playerHasStaticItemInAppearanceInventory(player, JEDI_CLOAK_LIGHT_HOOD_UP) || utils.playerHasStaticItemInAppearanceInventory(player, JEDI_CLOAK_DARK_HOOD_UP) || utils.playerHasStaticItemInAppearanceInventory(player, JEDI_CLOAK_LIGHT_HOOD_DOWN) || utils.playerHasStaticItemInAppearanceInventory(player, JEDI_CLOAK_DARK_HOOD_DOWN)) ? true : false;
```
Comment (Cekis) [inline sku.0/sys.server/compiled/game/script/library/jedi.java]: This is "shorter", but causes difficulty when trying to read it.  I would leave it as is or at least make the change multiple lines broken by the logical OR's.

**Comment by Cekis** [comment]
You indicated there were "Error Fixes" in this commit.  What errors did this fix?  I only see an attempt to optimize the code slightly.

File: sku.0/sys.server/compiled/game/script/library/jedi.java (hunk 1 above)
Comment (ResurgenceDark) [reply sku.0/sys.server/compiled/game/script/library/jedi.java]: Gotcha.  Thanks, I'll keep that in mind.

File: sku.0/sys.server/compiled/game/script/library/jedi.java (hunk 3 above)
Comment (ResurgenceDark) [reply sku.0/sys.server/compiled/game/script/library/jedi.java]: I guess I just didn't see it as difficult to read.  I just look at it as a reduction in lines of code.  Seemed worth doing, but thats me.

**Comment by ResurgenceDark** [comment]
Yeah, as I said in DM's, that was my fault.  I realized after the fact, it didn't fix anything, was just a cleanup.

**Pull request closed without merging**